libraries/AP_Scripting/examples/efi_gen_auto_switch.lua

```diff
@@ -0,0 +1,146 @@
+--[[
+    Switch EFI Generator on/off based on reported Load
```

Comment:
explain the odd behavior

Reply:
Essentially the Partner this is for says that the generator is only used to power payloads. So when the EFI engine is under heavy load, he'd rather not want the power to deviate.